src/Symfony/Component/Intl/Countries.php

```diff
@@ -65,6 +104,20 @@ public static function getNames($displayLocale = null)
         return self::asort(self::readEntry(['Names'], $displayLocale), $displayLocale);
     }
 
+    /**
+     * @return string[]
```

Comment:
this needs more documentation to document what it returns. This returns an array of names indexed by alpha3 code, which is not obvious from the method name.

Reply:
done